load.php

```diff
@@ -108,7 +108,23 @@ function perflab_get_module_settings() {
 	// Even though a default value is registered for this setting, the default must be explicitly
 	// passed here, to support scenarios where this function is called before the 'init' action,
 	// for example when loading the active modules.
-	return (array) get_option( PERFLAB_MODULES_SETTING, perflab_get_modules_setting_default() );
+	$perflab_get_module_settings = (array) get_option( PERFLAB_MODULES_SETTING, perflab_get_modules_setting_default() );
```

Comment:
Let's use a bit of a simpler variable name.
```suggestion
	$module_settings = (array) get_option( PERFLAB_MODULES_SETTING, perflab_get_modules_setting_default() );
```